Client/game_sa/CHandlingManagerSA.cpp

```diff
@@ -208,102 +216,88 @@ CHandlingManagerSA::~CHandlingManagerSA()
 
 eHandlingProperty CHandlingManagerSA::GetPropertyEnumFromName(std::string strName)
 {
-    std::map<std::string, eHandlingProperty>::iterator it;
-    it = m_HandlingNames.find(strName);
-
-    if (it != m_HandlingNames.end())
-    {
-        return it->second;
-    }
-    return HANDLING_MAX;
+    const auto it = m_HandlingNames.find(strName);
+    return it != m_HandlingNames.end() ? it->second : HANDLING_MAX;
 }
 
 CHandlingEntry* CHandlingManagerSA::CreateHandlingData()
 {
-    CHandlingEntrySA* pHandlingEntry = new CHandlingEntrySA();
-    return pHandlingEntry;
+    return new CHandlingEntrySA;
 }
 
 CFlyingHandlingEntry* CHandlingManagerSA::CreateFlyingHandlingData()
 {
-    CFlyingHandlingEntrySA* pFlyingHandlingEntry = new CFlyingHandlingEntrySA();
-    return pFlyingHandlingEntry;
+    return new CFlyingHandlingEntrySA;
 }
 
 CBoatHandlingEntry* CHandlingManagerSA::CreateBoatHandlingData()
 {
-    CBoatHandlingEntrySA* pBoatHandlingEntry = new CBoatHandlingEntrySA();
-    return pBoatHandlingEntry;
+    return new CBoatHandlingEntrySA;
 }
 
 CBikeHandlingEntry* CHandlingManagerSA::CreateBikeHandlingData()
 {
-    CBikeHandlingEntrySA* pBikeHandlingEntry = new CBikeHandlingEntrySA();
-    return pBikeHandlingEntry;
+    return new CBikeHandlingEntrySA;
 }
 
 const CHandlingEntry* CHandlingManagerSA::GetOriginalHandlingData(eVehicleTypes eModel)
 {
-    // Within range?
-    if (eModel >= 400 && eModel < VT_MAX)
+    // Vehicle?
+    if (IsVehicleModel(eModel))
     {
-        // Get our Handling ID
-        eHandlingTypes eHandling = GetHandlingID(eModel);
+        // Get our Handling ID, the default value will be HT_LANDSTAL
+        const auto eHandling = GetHandlingID(eModel);
         // Return it
         return m_pOriginalEntries[eHandling];
     }
-
-    return NULL;
+    return nullptr;
```

Comment:
Early return

Reply:
Excuse me, by this do you mean that after the condition, immediately make a return?

Reply:
yes
```cpp
{
    // Vehicle?
    if (!IsVehicleModel(eModel))
        return nullptr;

    // Get our Handling ID, the default value will be HT_LANDSTAL
    const auto eHandling = GetHandlingID(eModel);
    // Return it
    return m_pOriginalEntries[eHandling];
}
```